packages/react/README.md

```diff
@@ -7,21 +7,26 @@ Signals is a performant state management library with two primary goals:
 
 Read the [announcement post](https://preactjs.com/blog/introducing-signals/) to learn more about which problems signals solves and how it came to be.
 
-## Installation:
-
-```sh
-npm install @preact/signals-react
-```
-
-- [Guide / API](../../README.md#guide--api)
-  - [`signal(initialValue)`](../../README.md#signalinitialvalue)
-    - [`signal.peek()`](../../README.md#signalpeek)
-  - [`computed(fn)`](../../README.md#computedfn)
-  - [`effect(fn)`](../../README.md#effectfn)
-  - [`batch(fn)`](../../README.md#batchfn)
-  - [`untracked(fn)`](../../README.md#untrackedfn)
+- [Core API](../core/README.md#guide--api)
+  - [`signal(initialValue)`](../core/README.md#signalinitialvalue)
+    - [`signal.peek()`](../core/README.md#signalpeek)
+  - [`computed(fn)`](../core/README.md#computedfn)
+  - [`effect(fn)`](../core/README.md#effectfn)
+  - [`batch(fn)`](../core/README.md#batchfn)
+  - [`untracked(fn)`](../core/README.md#untrackedfn)
 - [React Integration](#react-integration)
+  - [Babel Transform](#babel-transform)
+  - [`useSignals` hook](#usesignals-hook)
```

Comment:
Could maybe do with a dedicated 'Installation' section here, but this reflects what's already on the page